university-service/build.gradle.kts

```diff
@@ -84,3 +102,28 @@ tasks.withType<Detekt>().configureEach {
 tasks.withType<DetektCreateBaselineTask>().configureEach {
     jvmTarget = "17"
 }
+
+protobuf {
```

Comment:
these tasks should be moved to protobuf module

Reply:
ok